### PR TITLE
some simple docs for cloud EDB

### LINKED ISSUE #337: Better docs for EDB / MongoDB cloud
MongoDB Clound Atlas instance

### PR DESCRIPTION
**Pull request by dangunter**
## Fixes/Addresses:

#337 

## Changes proposed in this PR:
- Add better docs

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by dangunter** [comment]
In some of the how-tos there are instructions for building IDAES config blocks manually. I added a note that these could be done with the EDB -- which I think is true? -- though it would be far nicer if we actually showed how to do that.

**Comment by dangunter** [comment]
fyi, I *was* seeing warnings (but really errors) in building the autodocs (for the source code) of the form:
```
WARNING: autodoc: failed to import module 'zero_order' from module 'watertap.unit_models'; the following exception was raised:
cannot import name 'InitializationError' from 'idaes.core.util.exceptions' (C:\Users\Dan Gunter\miniconda3\envs\watertap\lib\site-packages\idaes\core\util\exceptions.py)
```

I fixed the above by upgrading from idaes-pse 1.12.0 to 1.12.1

**Comment by aladshaw3** [comment]
@dangunter  You are still getting one more error on the docs

`/home/docs/checkouts/readthedocs.org/user_builds/watertap/checkouts/338/docs/how_to_guides/how_to_use_EDB.rst:19: WARNING: undefined label: install-mongo-local`

Looks like you are just missing a label somewhere in the new docs you added.

**Comment by aladshaw3** [comment]
> In some of the how-tos there are instructions for building IDAES config blocks manually. I added a note that these could be done with the EDB -- which I think is true? -- though it would be far nicer if we actually showed how to do that.

@dangunter 

There are some examples for this as jupyter notebooks (https://github.com/watertap-org/watertap/tree/main/docs/examples/edb). Those should already be a part of the documentation

File: docs/how_to_guides/how_to_use_EDB.rst [hunk 1]
```diff
@@ -20,3 +11,46 @@ For information on how to install and set up the EDB, see :ref:`install-edb`.
     ../examples/edb/edb_example
     ../examples/edb/simple_acid_example
     ../examples/edb/multiple_phases_example
```
Comment (aladshaw3) [inline docs/how_to_guides/how_to_use_EDB.rst:13]: @dangunter 

These are the examples that demonstrate how to use EDB to automatically generate the IDAES configs.

File: docs/how_to_guides/how_to_use_EDB.rst [hunk 2]
```diff
@@ -20,3 +11,46 @@ For information on how to install and set up the EDB, see :ref:`install-edb`.
...
+
+.. _install-mongodb:
+
+Installation
+------------
+Before running any of the EDB example Jupyter Notebooks, you **must** have MongoDB :ref:`installed locally <install-mongo-local>` and you need to load the 'bootstrap' data.
```
Comment (aladshaw3) [inline docs/how_to_guides/how_to_use_EDB.rst]:
```suggestion
Before running any of the EDB example Jupyter Notebooks, you **must** have MongoDB :ref:`installed locally <install-mongodb-local>` and you need to load the 'bootstrap' data.
```

Comment (aladshaw3) [reply docs/how_to_guides/how_to_use_EDB.rst]: I believe this is the issue with the docs right now.

File: docs/how_to_guides/how_to_setup_simple_chemistry.rst [hunk 3]
```diff
@@ -3,6 +3,9 @@
 How to setup simple chemistry
 =============================
 
+.. note:: This page provides a manual approach to building an IDAES configuration dictionary.
+    The same result can be achieved by using the :doc:`Electrolyte Database (EDB)</technical_reference/edb/index>`.
+
```
Comment (aladshaw3) [inline docs/how_to_guides/how_to_setup_simple_chemistry.rst]:
```suggestion
.. note:: This page provides a manual approach to building an IDAES configuration dictionary.
    The same result can be achieved by using the :doc:`Electrolyte Database (EDB)</technical_reference/edb/index>`. 
    Examples of this are provided under :ref:`How to use EDB <how_to_use_edb>`.
```

**Review by aladshaw3 — CHANGES_REQUESTED** [review]
Looks good. My only suggestion is to provide a link to the examples for how the IDAES configs are constructed with EDB.

File: docs/how_to_guides/how_to_setup_simple_chemistry.rst (hunk 3 above)
Comment (dangunter) [reply docs/how_to_guides/how_to_setup_simple_chemistry.rst]: OK I will do that and re-request review

**Review by aladshaw3 — APPROVED** [review]
LGTM

**Review by srikanthallu — APPROVED** [review]

**Pull request merged**